lighthouse-core/scripts/cleanup-LHR-for-diff.js

```diff
@@ -26,6 +26,9 @@ function cleanAndFormatLHR(lhrString) {
   /** @type {LH.Result} */
   const lhr = JSON.parse(lhrString);
 
+  lhr.fetchTime = new Date(2021, 6, 23).toISOString();
```

Comment:
This is timezone sensitive.
```suggestion
  lhr.fetchTime = new Date(1627023600000).toISOString();
```

That should be fine